x/oracle/keeper/query_server.go

```diff
@@ -0,0 +1,47 @@
+package keeper
+
+import (
+	"context"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/provenance-io/provenance/x/oracle/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// QueryAddress returns the address of the module's oracle
+func (k Keeper) OracleAddress(goCtx context.Context, req *types.QueryOracleAddressRequest) (*types.QueryOracleAddressResponse, error) {
```

Comment:
I bet the linter is complaining about `req` not being used. You'll want to change it to `_`.